src/index.js

```diff
@@ -91,47 +222,219 @@ export default class OsmRequest {
+   * @param {number} [options.user] Specifies the creator of the returned notes by using a valid id of the user. Does not work together with the display_name parameter
+   * @param {string} [options.time] Can be a unique value T1 or two values T1, T2 comma separated. Find changesets closed after value T1 or find changesets that were closed after T1 and created before T2. In other words, any changesets that were open at some time during the given time range T1 to T2. Time format is anything that http://ruby-doc.org/stdlib-2.6.3/libdoc/date/rdoc/DateTime.html#method-c-parse can parse.
+   * @param {number} [options.open] Only finds changesets that are still open but excludes changesets that are closed or have reached the element limit for a changeset (50.000 at the moment). Can be set to true
+   * @param {number} [options.closed] Only finds changesets that are closed or have reached the element limit. Can be set to true
+   * @param {number} [options.changesets] Finds changesets with the specified ids
+   * @return {Promise}
+   */
+  fetchChangesets(options) {
+    return fetchChangesetsRequest(this.endpoint, options);
+  }
+
   /**
    * Create a shiny new OSM node element, in a JSON format
    * @param {number} lat
    * @param {number} lon
-   * @param {[object]} [properties] Optional, initial properties
+   * @param {Object} [tags] Optional, initial tags
+   * @param {string} [id] Optional, identifier for OSM element
    * @return {Object}
    */
-  createNodeElement(lat, lon, properties = {}) {
-    const element = {
-      osm: {
-        $: {},
-        node: [
-          {
-            $: {
-              lat: lat,
-              lon: lon
-            },
-            tag: []
-          }
-        ]
+  createNodeElement(lat, lon, tags = {}, id) {
+    const node = {
+      $: {
+        lat: lat,
+        lon: lon
       },
+      tag: [],
       _type: 'node'
     };
+    if (id !== undefined) {
+      node._id = id.toString();
+    }
 
-    element.osm.node[0].tag = Object.keys(properties).map(propertyName => ({
+    node.tag = Object.keys(tags).map(tagName => ({
       $: {
-        k: propertyName.toString(),
-        v: properties[propertyName].toString()
+        k: tagName.toString(),
+        v: tags[tagName].toString()
       }
     }));
 
-    return element;
+    return node;
   }
 
   /**
-   * Fetch an OSM element by its ID
+   * Create a shiny new OSM way element, in a JSON format
+   * @param {Array<string>} nodeOsmIds
+   * @param {Object} [tags] Optional, initial tags
+   * @param {string} [id] Optional, identifier for OSM element
+   * @return {Object}
+   */
+  createWayElement(nodeOsmIds, tags = {}, id) {
+    const way = {
+      $: {},
+      nd: nodeOsmIds.map(id => {
+        return {
+          $: {
+            ref: findElementId(id)
+          }
+        };
+      }),
+      tag: [],
+      _type: 'way'
+    };
+
+    if (id !== undefined) {
```

Comment:
Just to be paranoid, I prefer to test with `typeof`. Because `foo !== undefined` works but `foo === undefined` can throw exceptions.

So if you forget that, or another developer don't know that, a `===` could be added soewhere and blow the module.

If you use the `typeof` style everywhere, other developers should stick with the convention and don't make mistakes.